recipes/eggnog-mapper/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: eggnog-mapper
+  version: 1.0.0
```

Comment:
should be 1.0.1

You can use 

```
{% set version = "1.0.1" %}
```
in the first lines and then 

```
package:
    version: {{ version }}

source:
   url: https:...../{{ version }}.tar.gz
```

to avoid getting those mixed up.

Reply:
Ah, good catch and suggestion. I fixed that up.